shaazk/Broadcaster
# #169838569 fix ui errors

#### What does this PR do?
- Fix UI errors.
#### Description of Task to be completed? 
- improved page responsiveness
- add html and css pages for good user experience(navigation)
- fixed button errors in design and link
#### How should this be manually tested?
 Clone the repository to anywhere in your local machine.
   - Clone repo on your local machine, open the folder on your vs code , click on any html page and run on live server. you should be able to navigate to other pages swiftly by clicking on the links in the page
#### Any background context you want to provide?
- N/A
#### What are the relevant pivotal tracker stories?
- [#169838569](https://www.pivotaltracker.com/story/show/169838569)
